app/src/main/java/com/example/muvi/data/model/GenreItem.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.muvi.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class GenreItem(
```

Comment:
```suggestion
data class Genre(
```
tương tự với những chỗ khác